R/Neural_Network/program/Helper.py

```diff
@@ -4,7 +4,7 @@
 # jimmy.royer@analysisgroup.com
 # May 29, 2016
 
-from sknn.mlp import Classifier, Layer, Native
```

Comment:
This file isn't maintained here, so modifications to it won't be retained if the file is ever updated.

Reply:
I'd say this is worth keeping given the last update was ~5 years ago.